capture-traffic/capture-from-android.md

```diff
@@ -10,31 +10,45 @@ previous_url: /knowledge-base/configure-android, /get-started/mobile-traffic/con
 
 - Connect an Android device to the same network or use an Android emulator on the Fiddler Everywhere host machine. Ensure that the machine on which Fiddler Everywhere and the Android device run is discoverable on the same network.
 
-## Configuring Fiddler Everywhere Host
+>important It's common for many company networks to be restricted by security tooling or administrative policies. To capture remote traffic, ensure that your system administrators are not limiting the network discovery and usage, or consider using a public hotspot for your Fiddler host and remote device.
+
+
+## Capturing Remote Android Traffic (Automated)
+
+Fiddler Everywhere provides an automated guide to configure remote Android capturing through the following steps:
+
+1. Start Fiddler Everywhere. 
+
+1. Open the **Home** pane.
+
+1. Open the **Remote Devices** screen.
 
-1. Enable the remote connections in Fiddler Everywhere through **Settings** > **Connections** > **Allow remote computers to connect**.
+1. Open the **Android** tutorial and follow the instructions.
+
+The tutorial requires administrative privileges for the current user (on the Android device) to install Fiddler's CA (certificate authority) and to set manual proxy. Once the steps are executed, you can immediately capture HTTPS traffic from the remote Android device.
+
+
+## Capturing Remote Android Traffic (Manual Setup)
+
+### Configuring Fiddler Everywhere Host
+
+1. Enable the remote connections in Fiddler Everywhere through **Settings** > **Connections** > **Allow remote computers to connect**. Combine it with **Keep it ON after app restart** if you want to persist the setting for subsequent application startups.
 
 1. Check the local IP address of the Fiddler Everywhere application host. You can use [the connection status on the lower right-hand side]({%slug connections-section%}) to obtain the host IP address. Alternatively, you can get the IP address using a terminal command like  **ipconfig**/ **ifconfig** (depending on the OS). For demonstration purposes, let's assume that the local IP of the Fiddler Everywhere host machine is **192.168.100.50**. 
 
-1. Ensure that the Fiddler root certificate is installed and the [**Capture HTTPS traffic** option]({%slug decrypt-https-traffic%}) is enabled.
+1. Ensure that the [**Capture HTTPS traffic** option]({%slug decrypt-https-traffic%}) is enabled.
+
+>tip: If you only need to capture remote traffic, then there is no need to install the Fiddler CA on the Fiddler host machine. You only need to download and install the Fiddler CA on the remote device. Once the proxy configuration is in place, Fiddler Everywhere will automatically capture and decrypt the remote HTTPS traffic.
 
-## Configuring Android Devices
+### Configuring the Fiddler proxy on Android Device
 
 The following steps apply to real Android devices with access to the Internet through the same network as the Fiddler Everywhere host machine.
```

Comment:
Can we edit the steps below to be the same as we have them in the wizard. This way we'll have consistency and we'll not end-up with statements like: "It is working through the wizard, but it is not working when I'm doing it manually". 
For example, the first step in the wizard is to try to open Fiddler Everywhere's URL on your device (through the IP), while here the first step is to configure the device to go through the proxy. With the current way described in the documentation, in case the device does not have access to Fiddler Everywhere's host, all your connections on the device will fail.